lib/smart_contracts/qutil_info.dart

```diff
@@ -0,0 +1,4 @@
+class QutilInfo {
+  static const address =
+      "EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF";
```

Comment:
@ahmed-tarek-salem  Address is wrong. Should be 
EAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVWRF